bigtable/use-cases/fraudDetection/README.md

```diff
@@ -84,4 +84,58 @@ The current Cloud Bigtable instance configuration does not provide any replicati
+```
+export GOOGLE_APPLICATION_CREDENTIALS={YOUR CREDS PATH}
+export PROJECT_ID={YOUR PROJECT ID}
+```
+
+### Running steps
+
+**1) Build the infrastructure**
+```
+cd terraform
+terraform init
+terraform apply -var="project_id=$PROJECT_ID"
+```
+This builds the infrastructure shown above, populates Cloud Bigtable with customers’ demographics data, and populates Cloud Bigtable with customers’ historical data. It takes about 5-10 minutes to finish.
+It builds the following resources:
+
+| Resource                         | Resource Name                                                                                                                           |
+|----------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| Cloud Bigtable Instance          | tf-fd-instance                                                                                                                          |
+| Cloud Bigtable Table             | tf-fd-table                                                                                                                             |
+| Cloud Bigtable Column Family     | history, demographics                                                                                                                   |
+| Cloud Pubsub Input Topic         | tf-fd-pubsub-input-topic                                                                                                                |
+| Cloud Pubsub Output Topic        | tf-fd-pubsub-output-topic                                                                                                               |
+| Cloud Pubsub Output Subscription | tf-fd-pubsub-output-subscription                                                                                                        |
+| Google Storage Bucket            | tf-fd-bucket-{RANDOM\_ID}                                                                                                               |
+| Google Storage Objects           | 1.  Temp directory<br/> 2.Testing-data <br/>3. Training-data <br/> 4. Pretrained ML model                                               |
+    | VertexAI Model                   | tf-fd-ml-model                                                                                                                          |
+    | VertexAI Endpoint                | **Name:** tf-fd-ml-model-ep <br/> **Id:** Determined in runtime, stored in Scripts/ENDPOINT\_ID.output                                  |
+| Dataflow Load CBT                | **Name:** fd-load-cbt-pipeline<br/> **Description:** Creates a batch job that loads both historical and demographic data from GS to CBT |
+
+**Alternatively**, you can run the maven test which will run the terraform commands and build the infrastructure, read the testing datasets, send them to the Cloud Pubsub input topic and wait for the response.
```

Comment:
I see your point.

What I had in mind is this: The first path allows customers to run the commands one by one, allowing them to have better configurability of what is happening under the hood. The second path is the easiest where with just one command, everything will be created, tested, and then destroyed.

I think I will just keep the first path, as the second one is way too abstract.

Reply:
With the single command, you still are going to need to set some environment variables probably/ set up somethings, so that is something to be mindful of.

Also, if it is just going to create and then destroy everything while you run the command, that leaves little room for interacting with the resources or seeing what is going on, so perhaps it is not worth keeping?